lib/native/sensor.c

```diff
@@ -0,0 +1,34 @@
+#include "../util.h"
+
+int initialize_sensors(void) {
+    puts("Nothing to do to initialize sensors ...[ok].");
+    return 1;
+}
+
+int read_humidity(void) {
+    error(-1, 0, "Sensor not present or read_humidity not implemented for this board.");    
```

Comment:
This makes `make term` crash always on `native`. I propose to just put a notice, and add a shell command `set_constant_sensor_value` to ease testing (the sensor reading can be done as normal then and would return this value all the time)